src/main/java/pl/zz/sharefood/util/ValidatePageUtil.java

```diff
@@ -0,0 +1,16 @@
+package pl.zz.sharefood.util;
+
+import pl.zz.sharefood.exception.BadRequestException;
+
+public class ValidatePageUtil {
+
+    public static int validatePageNumberAndSize(int page, int size) {
```

Comment:
walidacja nie powinna zwracać wartości innej niż true/false

Reply:
I dopiero gdy jest False to rzucać wyjątek w Service ? 
AppConstants.MAX_PAGE_SIZE - przypisywanie wartości domyślnej - wtedy usunąć ?

Reply:
Na ten moment pomić wywołanie walidacji. Jak będzie front dopięty to zastanowimy się czy jest potrzebna